requirements.txt

```diff
@@ -17,6 +17,7 @@ requests==2.22.0
 retrying==1.3.3
 scrapinghub==2.2.1
 six==1.10.0
+toml==0.10.0
```

Comment:
pipenv isn't required to run the library, I don't think we need to add `toml` for the specific case